.github/workflows/issue-bot.yaml

```diff
@@ -32,3 +32,9 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Update issue status if the issue is closed as completed
+        if: github.event.action == 'closed' && contains(github.event.issue.body, 'closed as completed')
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:in-progress"  --remove-label "status:accepted"
```

Comment:
Sorry about the back and forth here, but maybe we should just remove these labels in https://github.com/PrefectHQ/prefect/pull/8334/files#diff-4f76a181788cd19641d356309d984a1f4d19a2cfaa81eae2ac271ff63d92a19dR13?

Reply:
Oh yeah that makes sense. There's nothing special about being 'closed as completed' anymore 😞

Reply:
I don't know that "Remove status labels on close is the most descriptive" name or if we want to call them out individually

Reply:
Good enough for me!